### PR TITLE
Add Filters

### PR DESCRIPTION
**Pull request by leandroalonso**
### To test

1. Run the Example app
2. Tap the filters icon
3. Play with some filters and apply them
4. The image should be updated with the filter

**Important**: running in the simulator can be slow or cause some weird image outputs. Better to try in a real device.

PR submission checklist:

- [x] I have considered adding unit tests where possible — I'm really not sure if it worths to test the `CIImage` implementation. Open to feedback. :)
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: Sources/Capabilities/Filters/Views/MediaEditorFiltersViewController.swift [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+        sepiaFilter?.setValue(ciImage, forKey: kCIInputImageKey)
+
+        guard let outputImage = sepiaFilter?.outputImage else {
+            return image
+        }
+        return UIImage(ciImage: outputImage)
+    }
+
+    static func initialize() -> MediaEditorFiltersViewController {
+        return UIStoryboard(name: "MediaEditorFilters", bundle: Bundle(for: MediaEditorFiltersViewController.self)).instantiateViewController(withIdentifier: "filtersViewController") as! MediaEditorFiltersViewController
```
Comment (emilylaguna) [inline Sources/Capabilities/Filters/Views/MediaEditorFiltersViewController.swift]: This line is pretty long, what about breaking it up into multiple segments?

File: Sources/MediaEditor.swift [hunk 2]
```diff
@@ -12,7 +12,10 @@ import UIKit
 */
 open class MediaEditor: UINavigationController {
     /// The capabilities do be displayed in the Media Editor. You can add your own capability here.
-    public static var capabilities: [MediaEditorCapability.Type] = [MediaEditorCropZoomRotate.self]
+    public static var capabilities: [MediaEditorCapability.Type] = [MediaEditorFilters.self, MediaEditorCropZoomRotate.self]
+
+    /// A CIContext to be shared among capabilities. If your app already have one, you can assign it here.
```
Comment (emilylaguna) [inline Sources/MediaEditor.swift]:
```suggestion
    /// A CIContext to be shared among capabilities. If your app already has one, you can assign it here.
```

File: Sources/MediaEditor.swift [hunk 3]
```diff
@@ -12,7 +12,10 @@ import UIKit
 */
 open class MediaEditor: UINavigationController {
     /// The capabilities do be displayed in the Media Editor. You can add your own capability here.
```
Comment (emilylaguna) [inline Sources/MediaEditor.swift]:
```suggestion
    /// The capabilities are displayed in the Media Editor. You can add your own capabilities here.
```

**Review by emilylaguna — CHANGES_REQUESTED** [review]
- Filters are looking good 👍
- I got `EXC_BAD_ACCESS` when running the tests on iPhone 8 running iOS 11.4:  https://d.pr/i/moOOAY
- Made some suggestions for comment text and reducing a long line

File: Sources/Capabilities/Filters/Views/MediaEditorFiltersViewController.swift (hunk 1 above)
Comment (leandroalonso) [reply Sources/Capabilities/Filters/Views/MediaEditorFiltersViewController.swift]: Done in 59876f46698be33b72a234ffa6c9464c37854cde

What do you think?

Comment (emilylaguna) [reply Sources/Capabilities/Filters/Views/MediaEditorFiltersViewController.swift]: 👍Works for me!

**Review by emilylaguna — APPROVED** [review]
👍LGTM!

**Pull request merged**